component.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>POOL.ui | Saurabh Kamboj</title>
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/mobile.layout.style.css">
+    <link rel="stylesheet" href="/prism.css">
+    <link rel="shortcut icon" href="/images/favicon-32x32.png" type="image/png">
+</head>
+
+<body>
+
+    <div class="sidebar" id="side-nav">
```

Comment:
Its just a small suggestion you can use the Semantic HTML tags like **"aside"** for sidebar, **main** for the content section and **footer** for the social links.

It will follow the semantic HTML and improves accessibility. Just thought of mentioning it.

Reply:
Actually to make the sidebar responsive on toggle I decided to use div instead of aside. But surely, will do that in the upcoming version.

Reply:
Ok no problem.